tools/e2etesting/TestEventProcessor/TestEventProcessor.Businesslogic/TelemetryValidator.cs

```diff
@@ -330,6 +331,24 @@ private Task Client_ProcessEventAsync(ProcessEventArgs arg)
             return Task.CompletedTask;
         }
 
+        private bool CheckRestartAnnouncement(dynamic json) {
+            if (json is JObject) {
+                try {
+                    var messageType = (string)json["MessageType"].Value;
+                    var messageVersion = (int)json["MessageVersion"].Value;
+
+                    if (messageType == "RestartAnnouncement" && messageVersion == 1) {
```

Comment:
>messageVersion == 1 [](http://example.com/codeflow?start=64&length=19)

do we need this check? if the version is different, isn't the message still a restart announcement?

Reply:
There should not be a different version as of right now. So I am checking that as well. But you are correct that value of message version is not very important.